tests/benchmarks/micro_benchmarks/test_ib_loopback_performance.py

```diff
@@ -0,0 +1,252 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Tests for ib-loopback benchmark."""
+
+import os
+import numbers
+import unittest
+from pathlib import Path
+from unittest import mock
+
+from superbench.benchmarks import BenchmarkRegistry, Platform, BenchmarkType
+from superbench.common.utils import network
+
+
+class IBLoopbackBenchmarkTest(unittest.TestCase):
+    """Tests for IBLoopbackBenchmark benchmark."""
+    def setUp(self):
+        """Method called to prepare the test fixture."""
+        if (len(network.get_ib_devices()) < 1):
+            # Create fake binary file just for testing.
+            os.environ['SB_MICRO_PATH'] = '/tmp/superbench'
+            binary_path = Path(os.getenv('SB_MICRO_PATH'), 'bin')
+            binary_path.mkdir(parents=True, exist_ok=True)
+            self.__binary_file = Path(binary_path, 'run_perftest_loopback')
+            self.__binary_file.touch(mode=0o755, exist_ok=True)
+
+    def tearDown(self):
+        """Method called after the test method has been called and the result recorded."""
+        if (len(network.get_ib_devices()) < 1):
+            self.__binary_file.unlink()
+
+    @mock.patch('superbench.common.utils.network.get_free_port')
+    @mock.patch('superbench.benchmarks.micro_benchmarks.ib_loopback_performance.get_numa_cores')
```

Comment:
these two functions should be able to run in test VM?

Reply:
I’not sure what the return values in test VM, so I mock them to get deterministic return values

Reply:
but `get_numa_cores` function is not tested in unit tests

Reply:
add a test for these 2 functions